packages-internal/test-utils/src/initMatchers.ts

```diff
@@ -1,4 +1,4 @@
-import chai from 'chai';
+import * as chai from 'chai';
```

Comment:
This may have implications for X. Could make sense to double check

Reply:
The only usage I see in x is a side-effect import of `import '@mui/internal-test-utils/init';`.
We'll have to manually verify everything is working as expected when chai is bumped to v6 on x.